app/src/main/java/org/oppia/app/profile/ProfileChooserFragmentPresenter.kt

```diff
@@ -69,11 +69,20 @@ class ProfileChooserFragmentPresenter @Inject constructor(
   private fun bindProfileView(binding: ProfileChooserProfileViewBinding, data: ProfileChooserModel) {
```

Comment:
`data` is never used.

Reply:
This is intentional. I added a @Suppress to it.